ublox/src/ubx_packets.rs

```diff
@@ -82,7 +82,7 @@ pub trait UbxPacketCreator {
 }
 
 /// Packet not supported yet by this crate
-#[derive(Debug)]
```

Comment:
r.e. upstreaming this - there are a handful of things we need to do first, like any reference to serde has to be gated behind a serde feature, double check the core/alloc/std stuff and things like that. But I feel like we can sort that out later (the crate doesn't see much development so we don't have to worry about keeping getting behind). we can just use this crate via git dependency in gnss-converters-private